tck/rest/src/main/java/org/eclipse/microprofile/metrics/test/MpMetricTest.java

```diff
@@ -801,7 +805,43 @@ public void testMultipleTaggedMetricsProm() {
                         "taggedTimer_seconds_sum{mp_scope=\"application\",number=\"two\",tier=\"integration\"} "))
                 .body(containsString(
                         "taggedTimer_seconds_max{mp_scope=\"application\",number=\"two\",tier=\"integration\"} "));
+    }
 
+    @Test
+    @RunAsClient
+    public void testApplicationContainsHelpMessageOnce() {
+        given().header("Accept", TEXT_PLAIN).when().get("/metrics/application")
```

Comment:
For metrics 5.x  `/metrics?scope=application`

See: https://download.eclipse.org/microprofile/microprofile-metrics-5.1.1/microprofile-metrics-spec-5.1.1.html#scopes

Reply:
@Channyboy thanks, I updated the PR could you take a look and let me know if it needs any other changes